ada_2_fhir-r4/cio/2.0.0/beschikbaarstellen_contraindicatie/fhir_instance/mci-MCI-zwangerschap-2.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Flag xmlns="http://hl7.org/fhir">
+   <id value="mci-MCI-zwangerschap-2"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/cio-MedicationContraIndication"/>
+   </meta>
+   <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationContraIndication.RegistrationDateTime">
+      <valueDateTime value="${DATE, T, Y, -29}T10:11:42+02:00"/>
+   </extension>
+   <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationContraIndication.RelationContraIndication">
+      <valueReference>
+         <reference value="Flag/mci-MCI-zwangerschap-1"/>
+         <type value="Flag"/>
+         <display value="Zwangerschap"/>
+      </valueReference>
+   </extension>
+   <identifier>
+      <system value="urn:oid:2.16.840.1.113883.2.4.3.11.999.26.1.2"/>
+      <value value="MCI-zwangerschap-2"/>
+   </identifier>
+   <status value="inactive"/>
```

Comment:
Snomed code mist?

Reply:
In `Flag` moet je gebruikmaken van de volgende terminologie: http://hl7.org/fhir/R4/valueset-flag-status.html (binding is required), dus de SNOMED-codes uit de dataset worden met een ConceptMap vertaald naar de terminologie uit FHIR. Aangezien het gaat om een equivalente mapping, is het niet nodig om de originele code uit de dataset hier mee te geven.